docs/datamap/mv-datamap-guide.md

```diff
@@ -105,7 +105,7 @@ EXPLAIN SELECT a, sum(b) from maintable group by a;
    property is inherited from parent table, which allows user to provide different tableproperties
    for child table
  * MV creation with limit or union all ctas queries is unsupported
- * MV datamap does not support Streaming
+ * MV does not support Streaming
```

Comment:
I think complete document has to be changed. Using mv instead of datamap

Reply:
Yes, I will change doc later in another PR